build/mixxx.py

```diff
@@ -208,13 +208,15 @@ def __init__(self, target, machine, build, toolchain, available_features):
         self.read_environment_variables()
 
         # Now that environment variables have been read, we can detect the compiler.
-        self.compiler_is_gcc = 'gcc' in self.env['CC']
-        self.compiler_is_clang = 'clang' in self.env['CC']
+        import subprocess
+        process = subprocess.Popen([self.env['CC'], '--version'], stdout=subprocess.PIPE)
+        (stdout, stderr) = process.communicate()
+        self.compiler_is_gcc = 'gcc' in stdout
```

Comment:
I haven't tested it yet, but if `in` is case-sensitive then this check won't work either without applying `.lower()` to the result.

```
$ cc --version
cc (GCC) 9.1.1 20190503 (Red Hat 9.1.1-1)
```

Reply:
> I haven't tested it yet, but if `in` is case-sensitive then this check won't work either without applying `.lower()` to the result.

Indeed! I have force-pushed a corrected version, using `.lower()`.